### ISSUE TITLE
[Question] How to disable gutter signs

### ISSUE THREAD

**Issue by idr4n** [tanvirtin/vgit.nvim#277]
Hi and thanks for your nice plugin.

I'm using Gitsigns for git signs in the gutter and would like to use your plugin mainly for staging and previewing hunks. I noticed that despite disabling `live_gutter`, there is still an extra margin created by your plugin in the gutter:

Git signs with your plugin:
<img width="66" alt="image" src="https://user-images.githubusercontent.com/20104703/172044814-085bdd5b-55c6-4ee7-901e-869ec8c4da6d.png">

Git signs without your plugin:
<img width="66" alt="image" src="https://user-images.githubusercontent.com/20104703/172044829-343b6166-fb9c-43fd-9972-3c66fab9ce40.png">

This is my current `VGit` setup:

```lua
require("vgit").setup({
	settings = {
		live_gutter = {
			enabled = false,
			edge_navigation = false,
		},
		live_blame = {
			enabled = false,
		},
		authorship_code_lens = {
			enabled = false,
		},
		scene = {
			diff_preference = "split",
		},
	},
})
```

Is there a way to disable or get rid of that space in the gutter?

Thanks in advance!

**Comment by edwarmv** [comment]
You could try changing the sign text to something like this "▎"

**Comment by duarm** [comment]
Don't think this is VGit's fault, I have live_gutter disabled and use gitsigns for that. I have no margin (1 px actually)
![pic-selected-230111-1403-44 png](https://user-images.githubusercontent.com/26150581/211869755-c303f43d-b3b0-450d-a707-6881f83559d6.png)

**Comment by ray-x** [comment]
With vgit enabled and `gutter` set to false
![image](https://user-images.githubusercontent.com/1681295/216477951-1de5afb1-e9b2-48d4-a158-d3c364959d9e.png)

Without vgit
![image](https://user-images.githubusercontent.com/1681295/216478019-dc3e7a3a-9c6b-4683-b8ab-63154b5eeb4b.png)

**Comment by idr4n** [comment]
So investigating this issue further, it seems like `vgit` uses the same sign names as the `gitsigns.nvim` plugin (i.e, `GitSignsAdd`, etc.), and therefore it redefines the signs for `gitsigns.nvim` even when `live_gutter` is not enabled.

I changed this behavior locally by renaming the sign definitions in `vgit.settings.signs` to something else, e.g., `VGitSignsAdd`. Maybe there is a better way to avoid the conflict with `gitsigns.nvim`, such as not having the signs defined in the first place when `live_gutter` is not enabled.

Another option, without making any changes in the code, is just to define the same signs in the `vgit` settings as the ones I have in `gitsigns.nvim`. So basically the gap I referred to when I opened this issue it is because `vgit` uses a different unicode chracter by default, `'┃'`. So setting all signs in `vgit` to `'▎'`, which doesn't add a gap, solves this issue partially for me as well.

I'm closing this issue therefore.
